Явно не ми е ден.

katzata opened the pull request:
Нещо не мога да се концентрирам а и както не съм пипал от доста време... направо си е изложих.
Много време отделих на първата задача и от там се прекарах.

zzeni commented on task1/style.css:
```diff
@@ -13,6 +13,7 @@ body {
   border: 1px solid lightblue;
   border-radius: 8px;
   padding: 16px;
+    transform: rotate3d(x,y,z,angle);
```
това няма как да работи, защото не си задала стойности за x, y, z и angle

за да завъртиш един елемент при mouseover, трябва да напишеш следното:

```css
li {
  transition-delay: 1s;
}
li:hover {
  transform: translateY(360deg);
}
```

zzeni commented on task1/style.css:
```diff
@@ -25,18 +26,42 @@ ul {
 ul > li {
   list-style-type: none;
-  padding: 20px;
+padding:0px;
```
това е излишно

zzeni commented on task1/style.css:
```diff
@@ -25,18 +26,42 @@ ul {
+        background-color: lightblue;
+        background-image: 6;
+    }
+    
```
липсва затваряща скоба на медия правилото

zzeni commented on task1/style.css:
```diff
@@ -25,18 +26,42 @@ ul {
+@media only screen and (max-width: 1000px) {
+    body {
+        background-color: lightblue;
+        background-image: 6;
```
на `background-image` се подава url , който да зададе картинка за фон. В случая логиката ти е грешна.

За да зададеш фон само на четните позиции, се използва псевдо клас за селектор:

```css
li:nth-child(even) {
  background-color: lightblue;
}
```

zzeni commented on task1/style.css:
```diff
@@ -25,18 +26,42 @@ ul {
+    }
+    
+    @media only screen and (max-width: 600px) {
+   subtask1 {
```
този селектор не е правилен. За да селект-неш по `id` трябва да използваш `#`, ето така:
`#subtask1`

katzata committed:
Явно не ми е ден.

Нещо не мога да се концентрирам а и както не съм пипал от доста време... направо си е изложих.

zzeni commented:
Да, така става като мине много време, после е по-трудно.
Аз пък трябваше да ти кажа да не започваш от първата задача, защото тя наистина е трудничка.

Така или иначе, ти се справяше много добре през курса и научи цял куп неща. Сигурна съм, че ако решиш да се занимаваш по-сериозно с Уеб - ще се справиш.

Имаш 20 точки тук. Крайната ти оценка е 46% (71 - теория и 20 практика).
Ще пиша в академията да ти напечатат сертификат, за да можеш да си го вземеш.

Поздрави!

katzata commented:
Привет!  
 Объркването ми с първа задача идва от самия лист. Да накарам лист да се държи така ми е за първи път. Иначе бих го направил с flexbox и flex-grow за по лесно. Много надявам да си права за "заложбите" защото въпреки моментите в които ми се иска строша клавиатурата (не са малко) като цяло ми е приятно, не ме товари.  За сертификата да чакам допълнителен мейл или, след определен период да мина през академията?  Много благодаря за всичко и най-вече за всичките трикове които ни показа.  







 >-------- Оригинално писмо --------

 >От: Evgenia Manolova notifications@github.com

 >Относно: Re: [zzeni/final-test] Явно не ми е ден. (#1)

 >До: zzeni/final-test  

 >Изпратено на: 12.05.2018 10:48



 
 
  
Да, така става като мине много време, после е по-трудно.
 Аз пък трябваше да ти кажа да не започваш от първата задача, защото тя наистина е трудничка.  
  
Така или иначе, ти се справяше много добре през курса и научи цял куп неща. Сигурна съм, че ако решиш да се занимаваш по-сериозно с Уеб - ще се справиш.  
  
Имаш 20 точки тук. Крайната ти оценка е 46% (71 - теория и 20 практика).
 Ще пиша в академията да ти напечатат сертификат, за да можеш да си го вземеш.  
  
Поздрави!  
   —
You are receiving this because you authored the thread.
Reply to this email directly,  view it on GitHub , or  mute the thread .
